src/model/TurboFeed.java

```diff
@@ -23,123 +18,149 @@
 public class TurboFeed implements Listable {
```

Comment:
`TurboFeed` is a bit of an ambiguous name. Maybe `TurboFeedItem`, or just `FeedItem`? We might need to evaluate which classes get the `Turbo*` prefix in future.

I'm not sure about implementing `Listable`. `getListName` only allows for a string representation, which is probably inadequate for what we need (arbitrary layouts). Also it's been around for a while and we might not need it any more.